portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,88 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>A Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <!--Header  -->
+      <h1 style="margin:0;padding:0; font-family:'Brush Script MT'; font-size:80px">
+          Anya's Portfolio
+      </h1>
+      <h2 style="margin:0;padding:0">A portfolio made for SPS2020</h2>
+      <hr>
+
+      
+      <p style="margin-top:30px">Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      
+      <!-- Intro: details --> 
+      <p>My name is Anya Ellis. I am an undergrad at the University of Illinois at Chicago (UIC). 
```

Comment:
I didn't know you went to UIC! I graduated from IIT a little further southeast!

Reply:
Yeah, I know a couple of people from IIT from conferences but I haven't visited the campus yet. I mostly just hang out in Pilsen and Little Italy.